controllers/postgressnapshot/postgressnapshot_controller.go

```diff
@@ -51,11 +51,10 @@ type PostgresSnapshotReconciler struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
 	k8sclient.Client
-	scheme            *runtime.Scheme
-	logger            *logrus.Entry
-	provider          providers.PostgresSnapshotProvider
-	ConfigManager     croAws.ConfigManager
-	CredentialManager croAws.CredentialManager
```

Comment:
May be missing from this PR but why are we removing the `CredentialManager` here and in the other controllers ? Have we tested this on a non STS cluster ?

Reply:
@austincunningham It was removed as a result from feedback on the original PR by Oleg. It's not being used at all in `postgressnapshot` and `redissnapshot` controllers. Here's the link to the comment: https://github.com/integr8ly/cloud-resource-operator/pull/216#pullrequestreview-680520734

Reply:
Thanks @cecobask yep that clears it up 
no credentials calls in the controller , calls for credentials are done at the provider level
/lgtm
/approve